widget_driver_generator/lib/src/widget_driver_generator.dart

```diff
@@ -6,17 +6,25 @@ import 'package:widget_driver_generator/src/code_providers/driver_provider_code_
 import 'package:widget_driver_generator/src/code_providers/provided_properties_interface_code_provider.dart';
 import 'package:widget_driver_generator/src/code_providers/test_driver_code_provider.dart';
 import 'package:widget_driver_generator/src/utils/code_writer.dart';
+import 'package:yaml/yaml.dart';
 
 import 'model_visitor.dart';
+import 'utils/default_return_value_helper.dart';
 import 'utils/package_info_provider.dart';
 
 /// Generates TestDrivers and WidgetDriverProviders based on annotations
 class WidgetDriverGenerator extends GeneratorForAnnotation<GenerateTestDriver> {
   final PackageInfoProvider _packageInfoProvider;
 
   WidgetDriverGenerator({
+    required BuilderOptions options,
```

Comment:
This change needs to be reflected here: https://github.com/bmw-tech/widget_driver/blob/master/widget_driver_generator/lib/builder.dart#L10

Reply:
seems like I forgot to include it into my commit, thanks for pointing it out, it's added now. I also verified again that generation works :)